### ISSUE TITLE
[REGRESSION] Required parentheses are missing in SL1

### ISSUE THREAD

**Issue by jackmcbarn** [CensoredUsername/unrpyc#28]
As a result of f983608c, the following code decompiles incorrectly in SL1:
```
screen testScreen:
    modal False
    vbox:
        text ('Some variable is %s' % someVar)
```
This is the result of decompiling it:
```
screen testScreen:
    modal False
    vbox:
        text 'Some variable is %s' % someVar
```
The decompiled output isn't valid: "expected a keyword argument, colon, or end of line."
```
    text 'Some variable is %s' % someVar
                               ^
```

**Comment by CensoredUsername** [comment]
Strange, this didn't happen in the tests I ran with it.
What version are you using to test this? It might be that simple_expression parsing changed. As far as I know it works fine in 6.17.

**Comment by jackmcbarn** [comment]
6.15.7.374

**Comment by CensoredUsername** [comment]
Ah. It seems this change dates back to a change in ren'py, 17 Jan 2014 in which operators became supported in simple_expressions. I will change the code so it will add parenthesis around the affected equations too.

**Comment by CensoredUsername** [comment]
Should be fixed by 8ed7bd0c102140cdd1e3af7b7a3d287b7acf4c7d
